src/components/LoginOptions/LoginOptions.stories.mdx

```diff
@@ -0,0 +1,272 @@
+      paymentOptions: [],
+      literals: {
+        previousText: {resolved: 'Previous page', value: ''},
+        beginText: {resolved: 'Begin form', value: ''},
+        changeText: {resolved: 'Change', value: ''},
+        confirmText: {resolved: 'Confirm', value: ''},
+      },
+      slug: 'test-form',
+      url: `<placeholder>`,
+      steps: [],
+      showProgressIndicator: true,
+      maintenanceMode: false,
+      active: true,
+      explanationTemplate: '<p>Toelichtingssjabloon...</p>',
+      submissionAllowed: 'yes',
+      requiredFieldsWithAsterisk: true,
+    }}
+    onFormStart={onFormStart}
+  />
+);
+
+<Canvas>
+  <Story
+    name="Functional"
+    decorators={[LiteralDecorator]}
+    play={async ({args, canvasElement}) => {
+      const canvas = within(canvasElement);
+      let anonymousStartButton = await canvas.getByRole('button');
+      await userEvent.click(anonymousStartButton);
+      await waitFor(() => expect(args.onFormStart).toHaveBeenCalled());
+    }}
+    argTypes={{
+      form: {table: {disable: true}},
+      onFormStart: {
+        action: true,
+      }
+    }}
+    args={{
+      beginText: 'Begin form (anonymous)',
+      loginRequired: false,
+      loginOptions: [
+        {
+          identifier: "digid",
+          label: "DigiD",
+          url: "#",
+          logo: {
+            title: "DigiD simulatie",
+            imageSrc: "/digid.png",
+            href: "https://www.digid.nl/"
+          },
+          isForGemachtigde: false
+        },
+        {
+          identifier: "eherkenning",
+          label: "eHerkenning",
+          url: "#",
+          logo: {
+            title: "eHerkenning",
+            imageSrc: "/eherkenning.png",
+            href: "https://www.eherkenning.nl/"
+          },
+          isForGemachtigde: false
+        },
+        {
+          identifier: "eidas",
+          label: "eIDAS",
+          url: "#",
+          logo: {
+            title: "eIDAS",
+            imageSrc: "/eidas.png",
+            href: "https://digital-strategy.ec.europa.eu/en/policies/eu-trust-mark"
+          },
+          isForGemachtigde: false
+        },
+        {
+          identifier: "digid_machtigen",
+          label: "DigiD Machtigen",
+          url: "#",
+          logo: {
+            title: "DigiD Machtigen",
+            imageSrc: "/digid.png",
+            href: "https://www.digid.nl/"
+          },
+          isForGemachtigde: true
+        },
+        {
+          identifier: "eherkenning_bewindvoering",
+          label: "eHerkenning bewindvoering",
+          url: "#",
+          logo: {
+            title: "eHerkenning bewindvoering",
+            imageSrc: "/eherkenning.png",
+            href: "https://www.eherkenning.nl/"
+          },
+          isForGemachtigde: true
+        },
+      ],
+      onFormStart: jest.fn(e => e.preventDefault()),
+    }}
+  >
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<Canvas>
+  <Story
+    name="No \'machtigen\' options"
```

Comment:
I added a story for the situation without machtigen options so that we have visual regression testing to ensure the title is absent too if there are none.